docs/source/command-reference.mdx

```diff
@@ -237,18 +237,36 @@ The default value for `source` is `"uplink"`.
 
 #### Transport configuration
 
-These fields are under the top-level `transport` key. The available fields depend on the value of the nested `type` key.
-The default value for `type` is `"stdio"`.
-
-| Type           | Option    | Type                | Default     | Description                                                                                                                   |
-| :------------- | :-------- | :------------------ | :---------- | :---------------------------------------------------------------------------------------------------------------------------- |
-| stdio          | `type`    | `"stdio"`           | \*          | Use standard IO for communication between the server and client                                                               |
-| SSE            | `type`    | `"sse"`             |             | Host the MCP server on the supplied configuration, using SSE for communication. Note: Deprecated in favor of `StreamableHTTP` |
-| SSE            | `address` | `IpAddr`            | `127.0.0.1` | The IP address to bind to                                                                                                     |
-| SSE            | `port`    | `u16`               | `5000`      | The port to bind to                                                                                                           |
-| StreamableHTTP | `type`    | `"streamable_http"` |             | Host the MCP server on the configuration, using streamable HTTP messages.                                                     |
-| StreamableHTTP | `address` | `IpAddr`            | `127.0.0.1` | The IP address to bind to                                                                                                     |
-| StreamableHTTP | `port`    | `u16`               | `5000`      | The port to bind to                                                                                                           |
+These fields are under the top-level transport key, to configure running the MCP Server in different environments - `STDIO`, `SSE` or `StreamableHTTP`.
```

Comment:
```suggestion
These fields are under the top-level `transport` key and configure the MCP Server to run in different environments: stdio, Streamable HTTP, or SSE (deprecated).
```
- Use the same formatting as in the spec https://modelcontextprotocol.io/specification/2025-06-18/basic/transports
- Make note of deprecation as per spec